.github/workflows/oneDAL.yml

```diff
@@ -0,0 +1,116 @@
+name: oneDAL-nightly
```

Comment:
should we have this running in oneDAL repo and just deploy packages to https://github.com/oneapi-src/oneDAL/packages ?

Reply:
Good question, I am not sure which registry to use...  I mainly did it this way to simplify the next step (the testing CI), but I am open to changing it.